php/mailchimp.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+class MailChimpException extends Exception { };
+
+/**
+ * Class MailChimp
+ */
+
+class MailChimp {
+
+	private $api_uri;
+	private $api_user = 'api_v3';
+	private $api_key = '';
+	private $user_agent = 'API V3 - PHP Sample';
+
+	public function __construct($api_key)
+	{
+		$domain = explode('-', $api_key)[1];
+		$this->api_uri = 'https://' . $domain . '.api.mailchimp.com/3.0';
+		$this->api_key = $api_key;
+	}
+
+	private function _execute_request($method, $endpoint, Array $payload = [])
+	{
+		$endpoint = $this->api_uri.$endpoint;
```

Comment:
I'd suggest spaces around the concatenation operator to be consistent with code in other parts of this app:
```
$endpoint = $this->api_uri . $endpoint;
```